pkg/stub/handler.go

```diff
@@ -80,12 +81,26 @@ func (h *Handler) provisionNamespace(wt *v1alpha1.Walkthrough) (*v1alpha1.Walkth
 		},
 	}
 
-	namespace, err := h.k8sClient.CoreV1().Namespaces().Create(nsSpec)
+	_, err := h.k8sClient.CoreV1().Namespaces().Create(nsSpec)
 	if err != nil {
-		return nil, errors.Wrap(err, "failed to create wlakthrouh namespace")
+		return nil, errors.Wrap(err, "failed to create walkthrough namespace")
 	}
 
-	logrus.Debugf("namespace %+v", namespace)
+	wtCopy.Status.Phase = v1alpha1.PhaseUserRoleBindings
+	return wtCopy, nil
+}
+
+func (h *Handler) userRoleBindings(wt *v1alpha1.Walkthrough) (*v1alpha1.Walkthrough, error) {
+	wtCopy := wt.DeepCopy()
+
+	userRoles := []string{"view", "edit"}
```

Comment:
@maleck13 What roles do we want the user to have on this namespace?

Reply:
edit should be enough I think